### DISCUSSION TITLE
Extend "tpoffline" command to support updating last/logout location

### DISCUSSION THREAD

**Discussion by Sxtanna** [EssentialsX/Essentials#4867] [Ideas and Feature Suggestions]
Please extend this command to support other common teleport operations:
- tpofflinehere \<player\>
- tpoffline \<player\> \<target\:{online or offline}>
- tpoffline \<player\> \<x\> \<y\> \<z\>

**Comment by maxwellvalasek** [comment]
This would be a splendid addition

**Reply by yeatt** [reply]
Agreed!
